Client/src/Components/Game/Users/PlayersList/Player/PlayerCard.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+	import type Player from 'Models/Game/Player';
+
+	import strToGradient from 'Utils/strToGradient';
+
+	export let player: Player;
+
+	const playerColor = `background: ${strToGradient(player.username)}`;
+</script>
+
+<div class="grid grid-cols-4 p-2 text-lg wrapper">
+	<div class="col-span-1">
+		<div class="active-circle" style="{playerColor}"></div>
+	</div>
+	<div class="col-span-3">
+		{player.username}
```

Comment:
if the name is too long it leaves the card:
![image](https://user-images.githubusercontent.com/71587041/117415585-25545f00-af21-11eb-98eb-2bd2b98ecfdf.png)

Reply:
Do you want to put elipsis, limit the userName in the future, or do you have a different idea?

Reply:
I'll add ellipsis